components/list/list-item-drag-mixin.js

```diff
@@ -98,3 +99,84 @@ export const ListItemDragMixin = superclass => class extends superclass {
 		` : nothing;
 	}
 };
+
+export class NewPositionEventDetails {
+	/**
+	 * @param { Object } object An simple object with the position event properties
+	 * @param { String } object.targetKey The item key of the list-item that is moving
+	 * @param { String } object.destinationKey The item key of the list-item in the position we are moving to
+	 * @param { String } object.temporaryMovement Information on whether the item is entering or exiting temporary movement
+	 */
+	constructor(object) {
```

Comment:
Can you do `constructor({targetKey, destinationKey, temporaryMovement})` doing the cool es6 stuff that we see in our services.